src/Core/functions.php

```diff
@@ -199,7 +199,7 @@ function env($key, $default = null)
                 return (!empty($_SERVER['HTTPS']) && $_SERVER['HTTPS'] !== 'off');
             }
 
-            return (strpos(env('SCRIPT_URI'), 'https://') === 0);
+            return (strpos((string)env('SCRIPT_URI'), 'https://') === 0);
```

Comment:
Nope. You can pass a default value as second parameter bit this defaults to null.